org.openhab.binding.zigbee/README.md

```diff
@@ -18,6 +18,18 @@ demo.things:
 Thing zigbee:coordinator_cc2531:stick1 "Zigbee USB Stick" [zigbee_port="/dev/ttyACM0", zigbee_baud="38400"]
 ```
 
+#### Serial port Configuration
+
+If you are using a config file, the serial port needs to be configured as shown in the above example. Below is a list of the relevat settings concerning the serial port.
+
+| Setting               | Allowed values  |
+| --------------------- | --------------- |
+| `zigbee_baud`         | `38400`, `57600`, `115200` |
```

Comment:
This is not correct. The allowable values are not really defined and are dependant on what coordinator you're using.